src/Pages/ChangePassword.cshtml.cs

```diff
@@ -56,5 +64,17 @@ public async Task<IActionResult> OnPost()
                 return Page();
             }
         }
+
+        public async Task<IActionResult> OnPostDeleteAccount()
+        {
+            await HttpContext.SignOutAsync();
+            await UserDatabase.DeleteUserAsync(User.FindFirst(ClaimTypes.Email).Value);
+            return Redirect("/index");
+        }
```

Comment:
Maybe we should delete all data associated with the user here as well so we don't clog up the database with data that does belong to inactive users

Reply:
I think more logical approach would so the method `DeleteUserAsync` would call such method to delete data, since it would protect from putting data inside front.

Reply:
That makes a lot more sense, I've implemented this into this branch, and I think this is ready to be approved

Reply:
Glad to hear. I guess, yeah, since when I tested it worked I think I'll merge this one.